templates/docker-monolithic/restore.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -eu -o pipefail
+
+LAST_BACKUP_DIR="backup/$(ls -1t backup/ | head -n1)"
+LAST_BACKUP_FILE=$(ls -1 ${LAST_BACKUP_DIR} | grep 'app-.*gz')
+
+
+TIMESTAMP=$(date +%Y-%m-%d-%H-%M-%S-%s)
+TEMP_BACKUP_DIR="/tmp/tmp-backups/${TIMESTAMP}"
+docker exec supportpal bash -c "mkdir -p ${TEMP_BACKUP_DIR}"
+docker cp "${LAST_BACKUP_DIR}/${LAST_BACKUP_FILE}" "supportpal:${TEMP_BACKUP_DIR}/"
+docker exec supportpal bash -c "cd /var/www/supportpal && php artisan app:restore ${TEMP_BACKUP_DIR}/${LAST_BACKUP_FILE} --no-verify --force"
```

Comment:
Should this be ran as `www-data` to suppress:
```
You are running the command as "root", while "/var/www/supportpal" is owned by "www-data", this can lead to file permission issues.
```

The backup still seems to work correctly as I guess `tar` restores the user/group/permissions from the backup.

Reply:
Results in permission issues.